src/cplscheme/CompositionalCouplingScheme.hpp

```diff
@@ -141,33 +141,33 @@ class CompositionalCouplingScheme final : public CouplingScheme {
   int getTimeWindows() const final override;
 
   /**
-   * @brief Returns true, if timestep length by any of the coupling schemes in this compositional coupling scheme.
+   * @brief Returns true, if time window size is given by any of the coupling schemes in this compositional coupling scheme.
```

Comment:
Another tricky use of time step vs window. I would argue: a given time *window* size counts as a limit for the time *step* in a compositional coupling scheme.

Reply:
In the config we use:
```xml
<time-window-size method="first-participant"/>
```
-> I would also use "time window" here.